tests/test_windows_utils.py

```
@@ -226,6 +294,158 @@ def test_test_config_generation_windows_handles_exceptions(
         self.assertEqual(result["overall_status"], "partial")
         self.assertEqual(result["sample_config_path"]["status"], "error")
 
+    def test_test_config_generation_windows_sample_path_missing(self):
+        """Missing sample config path should mark sample test as error."""
+        with (
+            patch("mmrelay.windows_utils.is_windows", return_value=True),
+            patch(
+                "mmrelay.tools.get_sample_config_path",
+                return_value="/tmp/missing_sample.yaml",
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/missing_sample.yaml"

(S108)